lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -987,6 +993,14 @@ def test_courseware_data(self, mock_request):
         self.assertEqual(response_data['discussion_data'][0]['courseware_url'], expected_courseware_url)
         self.assertEqual(response_data["discussion_data"][0]["courseware_title"], expected_courseware_title)
 
+    def test_courseware_data(self, mock_request):
+        self.verify_response(self.send_request(mock_request))
+
+    def test_context(self, mock_request):
+        response = self.send_request(mock_request, {'context': 'standalone'})
+        self.assertEqual(mock_request.call_args[1]['params']['context'], 'standalone')
```

Comment:
There's not much to test here because it runs against a mock comments service. What this test checks is that if you pass a context to the inline discussion view handler (`{'context': 'standalone'}`), that same context is present in the request issued to the comments service (`mock_request.call_args[1]['params']['context'] == 'standalone'`). It's just a test that the context value is correctly threaded through the LMS discussion API.